app/controllers/support_interface/candidates_controller.rb

```diff
@@ -18,6 +17,8 @@ def index
         candidate_number = @filter.applied_filters[:candidate_number].tr('^0-9', '')
         @candidates = @candidates.where(id: candidate_number)
       end
+
+      @pagy, @candidates = pagy(@candidates, page: params[:page], items: 30)
```

Comment:
Is the `page: params[:page]` argument needed? If so is it needed in other places too or is it implied everywhere?